bedrock/base/templates/includes/structured-data/organization/mozilla-foundation-organization.json

```diff
@@ -0,0 +1,31 @@
+{
+    "@context": "http://schema.org",
+      "@type": "NGO",
+      "@id": "https://foundation.mozilla.org/#organisation",
+      "name": "Mozilla Foundation",
+      "url": "https://foundation.mozilla.org",
+      "sameAs": [
+        "https://www.wikidata.org/wiki/Q55672",
+        "https://foundation.mozilla.org",
+        "https://en.wikipedia.org/wiki/Mozilla_Foundation"
+      ],
+      "subOrganization": {
+        "@type": "Organization",
+		"@id": "https://www.mozilla.org/#organization",
+    	"name": "Mozilla",
+    	"alternateName": "MoCo",
+    	"legalName": "Mozilla Corporation",
+    	"url": "https://www.mozilla.org",
+    	"logo": "https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg",
+    	"sameAs": [
+      	"https://www.wikidata.org/wiki/Q169925",
+      	"https://twitter.com/mozilla",
+      	"https://www.facebook.com/mozilla",
+      	"https://github.com/mozilla",
+      	"https://www.instagram.com/mozilla/",
+      	"https://www.youtube.com/user/Mozilla",
+      	"https://en.wikipedia.org/wiki/Mozilla_Corporation",
+      	"https://www.linkedin.com/company/mozilla-corporation"
+    	]
+      }
+  }
```

Comment:
Build out of pure curiosity, but wondering if having a link checker for all these URLs would be useful.
The one huge caveat is that we'd need to know what schema each JSON file has to determine where to look for URLs (which may be more trouble than it's worth):

```js
const fs = require("fs");
const checkLinks = require("check-links");

// See https://medium.com/javascript-inside/safely-accessing-deeply-nested-values-in-javascript-99bf72a0855a
const get = (p, o) => p.reduce((xs, x) => (xs && xs[x]) ? xs[x] : null, o);

main();

async function main() {
  const json = loadJsonSchema("./bedrock/base/templates/includes/structured-data/organization/mozilla-foundation-organization.json");
  const properties = [
    "url",
    "sameAs",
    "subOrganization.url",
    "subOrganization.logo",
    "subOrganization.sameAs"
  ];
  const links = getProps(properties, json);
  const res = await checkLinks(links);

  const brokenLinks = Object.entries(res)
    .reduce((arr, [url, res]) => arr.push(Object.assign({url}, res)) && arr, [])
    .filter(link => link.status !== "alive");
  console.log(brokenLinks);
}

function getProps(properties, json) {
  // NOTE: `Array#flat()` requires Node 12+.
  const res = properties.map(prop => get(prop.split("."), json)).flat();
  // Use a temporary `Set()` to remove potential duplicate URLs.
  return [...new Set(res)].sort();
}

function loadJsonSchema(p) {
  return require(p);
}
```